tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/DevOpsService.cs

```diff
@@ -537,5 +560,249 @@ public static string ParseSDKPullRequestUrl(string sdkGenerationSummary)
             var match = regex.Match(sdkGenerationSummary);
             return match.Success ? match.Value : string.Empty;
         }
+
+        /// <summary>
+        /// Add the list of SDK languages to release plan. Release plan uses this language list to track SDK release.
+        /// </summary>
+        /// <param name="workItemId"></param>
+        /// <param name="sdkLanguages"></param>
+        /// <param name="languageExclusionNote"></param>
+        /// <returns>bool</returns>
+        /// <exception cref="ArgumentException"></exception>
+        /// <exception cref="Exception"></exception>
+        public async Task<bool> UpdateReleasePlanSDKDetails(int workItemId, List<SDKInfo> sdkLanguages, string languageExclusionNote = "")
+        {
+            // Adds SDK languages in release plan work item.
+            try
+            {
+                if (workItemId == 0 || sdkLanguages == null || sdkLanguages.Count == 0)
+                {
+                    throw new ArgumentException("Please provide the work item ID and a list of languages to add SDK info to a work item.");
+                }
+
+                HashSet<string> languageNames = [.. sdkLanguages.Select(s => s.Language)];
+                var releasePlan = await GetReleasePlanForWorkItem(workItemId);
+                if (releasePlan?.SDKInfo != null)
+                {
+                    languageNames.UnionWith(releasePlan.SDKInfo.Select(s => s.Language));
+                }
+
+                // Make sure all languages are supported or exception note is provided
+                if (languageNames.Count < 5 && string.IsNullOrEmpty(languageExclusionNote))
```

Comment:
this will work for data plane, but not for management plane.
Requirement is:
- Management: .NET, Go, Java, Javascript, and Python
- Data: .NET, Java, Javascript, and Python

Reply:
Oh, I can make that change.